app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx

```diff
@@ -2,12 +2,20 @@ import React from "react";
 
 import { Formik, Form } from "formik";
 import { Button, Pane, Toastr } from "neetoui/v2";
-import { Input, Textarea } from "neetoui/v2/formik";
+import { Input, Select, Textarea } from "neetoui/v2/formik";
+import { Check } from "neetouiIcons";
 
 import notesApi from "apis/notes";
 import formValidationSchemas from "constants/formValidationSchemas";
 
-export default function NoteForm({ onClose, refetch, note, isEdit }) {
+export default function NoteForm({
+  onClose,
+  refetch,
+  note,
+  isEdit,
+  contacts,
+  tags,
+}) {
   const handleSubmit = async values => {
```

Comment:
Please move `NOTE_VALIDATION_SCHEMA` and `NOTE_INTIAL_VALUES` to a constant file in the scope of `Notes` directory.